static/app/utils/replays/hooks/useExtractedCrumbHtml.tsx

```diff
@@ -0,0 +1,120 @@
+import {useEffect, useState} from 'react';
+import first from 'lodash/first';
+import {Replayer} from 'rrweb'; // , ReplayerEvents
+import {eventWithTime} from 'rrweb/typings/types';
+
+import type {Crumb} from 'sentry/types/breadcrumbs';
+import type ReplayReader from 'sentry/utils/replays/replayReader';
+
+const IncrementalSnapshot = 3;
+
+type Extraction = {
+  crumb: Crumb;
+  html: string;
+  timestamp: number;
+};
+
+type HookOpts = {
+  domRoot: null | HTMLDivElement;
```

Comment:
I think i can remove this opt and just let the hook create it's own hidden div on the `<body>` instead.

Reply:
done